samples/copy-from-container/README.md

```diff
@@ -0,0 +1,64 @@
+# Docker Maven Plugin Copy Sample
+
+This is a plain maven application used to demonstrate how to copy files or directories from container using `docker:copy` goal
+
+## How to Build?
```

Comment:
To be honest I don't see a need in this section, because `package` does nothing for this sample.